data/io.elementary.terminal.gschema.xml

```diff
@@ -103,7 +103,7 @@
       </description>
     </key>
     <key name="background" type="s">
-      <default>"rgba(37, 46, 50, 0.95)"</default>
```

Comment:
The `.color-dark radio` in Application.css still uses this color (with 100% opacity), should probably be changed to the gray color as well?

Reply:
@jlnr good catch, addressed.